Uploader/Storage/FilesystemOrphanageStorage.php

```diff
@@ -68,15 +89,15 @@ public function getFiles()
             $finder->append([]);
         }
 
-        return $finder;
+        return iterator_to_array($finder);
```

Comment:
Hello

I think getFiles() need to return a Finder instance (like before) instead of Array; or doc needs to be updated, because:

       
>         // reduce the scope of the Finder object to what you want
>         $files->files()->name($filename);
>         $manager->uploadFiles(iterator_to_array($files));

In this scenario, $files needs to be à Finder object.


I use Orphanage in my project, if getFiles() return an array, i need to update my code, but it's add complexity:

```
		/** @var $productOrphaneManager \Oneup\UploaderBundle\Uploader\Storage\FilesystemOrphanageStorage **/
		$productOrphaneManager	= $OrphanageManager->get('products');
		$files			= $productOrphaneManager->getFiles();

			foreach ($request->get('bt_corebundle_product')['dropzone'] as $filename)
			{
				$files->files()->name($filename);
			}
			$productOrphaneManager->uploadFiles(iterator_to_array($files));
```

should be:
```
		/** @var $productOrphaneManager \Oneup\UploaderBundle\Uploader\Storage\FilesystemOrphanageStorage **/
		$productOrphaneManager	= $OrphanageManager->get('products');
		$files			= $productOrphaneManager->getFiles();

			$imageToMove = [];
			foreach ($request->get('bt_corebundle_product')['dropzone'] as $filename)
			{
				foreach ($files as $file)
				{
					if ($file->getFileName() == $filename)
					{
						$imageToMove[] = $file;
					}
				}
			}

			$productOrphaneManager->uploadFiles($imageToMove);
```

Reply:
Hi

I've changed this primarly because the other orphanage storages (`FlysystemOrphanageStorage` and `GaufretteOrphanageStorage` return also arrays with `getFiles()`).

Reply:
Reverted in c287f14.